CHANGELOG.md

```diff
@@ -779,6 +897,38 @@ rebuilt upon changes to the list of issuers. [[GH-15179](https://github.com/hash
 * ui: fix search-select component showing blank selections when editing group member entity [[GH-15058](https://github.com/hashicorp/vault/pull/15058)]
 * ui: masked values no longer give away length or location of special characters [[GH-15025](https://github.com/hashicorp/vault/pull/15025)]
 
+## 1.10.10
+### January 5, 2023
+
+CHANGES:
+
+* core: Bump Go version to 1.19.4.
+
+FEATURES:
```

Comment:
Should be removed (note to self, since I'm going to close the PR and open a new one instead).